.circleci/config.yml

```diff
@@ -4,10 +4,11 @@ jobs:
 
   build:
     environment:
+      JAVA_HOME: "/usr/local/jdk-11"
       _JAVA_OPTIONS: "-Xms512m -Xmx1g"
     working_directory: ~/workspace
     docker:
-      - image: circleci/openjdk:11-jdk
+      - image: cimg/openjdk:11.0.0
```

Comment:
should the latest `cimg/openjdk:11.0.12` be used here? Or maybe `cimg/openjdk:11.0` to always get the latest jdk 11 patch version?

Reply:
Thanks @amuraru -- using minor release tag (i.e. `cimg/openjdk:11.0`) is a good idea. Looks like doing so will indeed automatically point to future patch updates as they are released by AdoptOpenJDK & Oracle (source: https://circleci.com/developer/images/image/cimg/openjdk)

Reply:
Latest job gets java `11.0.12`. 
```
$ /usr/local/jdk-11.0.12/bin/java --version
Picked up _JAVA_OPTIONS: -Xms512m -Xmx1g
openjdk 11.0.12 2021-07-20
OpenJDK Runtime Environment Temurin-11.0.12+7 (build 11.0.12+7)
OpenJDK 64-Bit Server VM Temurin-11.0.12+7 (build 11.0.12+7, mixed mode)
```